orchagent/aclorch.cpp

```diff
@@ -1060,6 +1060,108 @@ void AclOrch::doTask(Consumer &consumer)
     }
 }
 
+void AclOrch::addAclTable(AclTable &newTable, string table_id)
```

Comment:
in this and other functions I would suggest to return some status and not void
it will be useful for the PFC WD or some other caller

Reply:
Returning the status is not necessary for now, since in aclorch we are not checking these status. We could refactor and add this later if we want to check the status in aclorch.

Reply:
@andriymoroz-mlnx I think it is okay that we update this in the future pull request. More refactorings are needed.